### PR TITLE
Partially Fixes #418 :- Media Tabs added

### LINKED ISSUE #418: Making Advanced search Navbar of feed results
Idea is to make advanced search navbar of feed results just like market leader (without hard coding). Moreover, adding options/filters and linking to different actions.

### PR DESCRIPTION
**Pull request by Achint08**
**Changes proposed in this pull request**
- Media Tabs added and working
-
-

**Screenshots (if appropriate)** 

**Link to live demo (if appropriate):** 

https://achint08.github.io/loklak_search/

**Closes #418 **

### CODE REVIEW

**Comment by Achint08** [comment]
Please review @hemantjadon @mariobehling @djmgit @vibhcool @singhpratyush @SKrPl @kavithaenair

File: src/app/feed/feed-advanced-search/feed-advanced-search.component.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+	<div class="tabslist">
+		<ul>
+			<li [class.active_view]="show('all')" class="tabs" (click)="getAllResults()">All</li>
```
Comment (hemantjadon) [inline src/app/feed/feed-advanced-search/feed-advanced-search.component.html]: Don't make this show method, you can handle this logic inline only `[class.active]="selectedTab==='all'"` no need to make this function call.

**Comment by Achint08** [comment]
@hemantjadon Done!

**Comment by Achint08** [comment]
@singhpratyush I have added an issue for this before. The Only solution is to remove .m3u8 files temporarily.

**Comment by singhpratyush** [comment]
> The Only solution is to remove .m3u8 files

But they can be ignored too, even if you are receiving as a response from the server.

**Comment by singhpratyush** [comment]
Also, videos start playing soon as the results are displayed.

**Comment by Achint08** [comment]
@singhpratyush Removed it. Checkout now. :)

**Comment by vibhcool** [comment]
https://achint08.github.io/loklak_search/search?query=car
see this, appears to be some error
this is html division of the box appearing
```
<div class="singleimage" _ngcontent-c5="">
<img class="card-image" _ngcontent-c5="" src="https://pbs.twimg.com/card_img/881689464954900480/tFS60G98?format=jpg&amp;name=280x280">
</div>
```

screenshot from 2017-07-07 16-58-36

File: src/app/feed/feed.module.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@ import { UserInfoBoxComponent } from './user-info-box/user-info-box.component';
 import { FeedLightboxComponent } from './feed-lightbox/feed-lightbox.component';
 import { FeedUserCardComponent } from './feed-user-card/feed-user-card.component';
 import { InViewportDirective } from '../shared//in-viewport.directive';
+import { FeedAdvancedSearchComponent } from './feed-advanced-search/feed-advanced-search.component';
```
Comment (vibhcool) [inline src/app/feed/feed.module.ts]: extra space...

File: src/app/feed/feed.module.ts [hunk 3]
```diff
@@ -44,6 +45,10 @@ import { InViewportDirective } from '../shared//in-viewport.directive';
 		ReactiveFormsModule,
 
 		/**
+		 * The ng module for Forms
```
Comment (vibhcool) [inline src/app/feed/feed.module.ts]: instead of multiline comment, you could use single-line comment

Comment (Achint08) [inline src/app/feed/feed.module.ts]: If you checkout the file, We use multi-lined comments so that more information can be added later.

**Comment by Achint08** [comment]
@vibhcool @singhpratyush @hemantjadon Please review now.

**Comment by Achint08** [comment]
@singhpratyush When we switch to `image` tab, a request is made to the server with the query and `filter=image`. However, to fix this, we need to filter out feeds with image content, I think @hemantjadon will be working on it.

**Comment by singhpratyush** [comment]
@Achint08: Can you please open an issue for this in https://github.com/loklak/loklak_server?

**Comment by Achint08** [comment]
@singhpratyush Sure. 👍

**Comment by mariobehling** [comment]
@Achint08 As we are all part of the server project, it would be great to see you working on a fix for the server to get this feature right. Cheers.

**Comment by mariobehling** [comment]
Search "customization" is actually just search, but you could add additional facets here. Before this features goes into the media wall it should go into an advanced search page like https://twitter.com/search-advanced. Please follow up in this issue here: https://github.com/fossasia/loklak_search/issues/447

**Pull request merged**